sharedTest/build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id 'com.android.library'
     id 'kotlin-android'
     id 'kotlin-kapt'
+    id 'org.jetbrains.kotlin.android'
```

Comment:
This plugin id was deprecated in favour of kotlin-android plugin id. Why add this?

Reply:
Automatic kotlin upgrade by ide added this, i thought i removed everything but looks like this is still here. Will be removing this.